modules/view-compliance-status-for-standard.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-compliance.adoc
+:_module-type: PROCEDURE
+[id="view-compliance-status-for-standard_{context}"]
+= View compliance status for a specific standard
+
+[role="_abstract"]
+{product-title} supports NIST, PCI DSS, NIST, HIPAA, CIS for Kubernetes and CIS for Docker compliance standards.
+You can view all the compliance controls for a single compliance standard.
+
+.Procedure
+. Navigate to the RHACS portal and open the compliance dashboard by selecting *Compliance* from the left-hand navigation menu.
+. On the compliance dashboard, look for the *Passing standards by cluster* widget.
+. In this widget, click on a *<standard>* to view information about all the controls associated with that standard.
```

Comment:
Not critical, but I don't think it's necessary to format this as replaceable/bold GUI when you're just saying to click on one of them. Same thing for the other modules.